yoga/YGMacros.h

```diff
@@ -49,7 +49,7 @@
 #define YG_ENUM_END(name)
 #else
 #define YG_ENUM_BEGIN(name) enum name
-#define YG_ENUM_END(name) name
+#define YG_ENUM_END(name) __attribute__((enum_extensibility(closed))) name
```

Comment:
A couple of these are flag enums, which would also need to be declared I think

Reply:
@NickGerleman do we want this for `YGErrata & YGExperimentalFeature` or are there any others?

Reply:
Just realised that enums.py specifies that only `Errata` is a bitset enum.

Reply:
That sounds good to me!

Reply:
@NickGerleman , I took a quick stab at this on [this branch](https://github.com/facebook/yoga/compare/main...iainsmith:yoga:flag_enums?expand=1), but I've run into a compiler error I'm not sure how to approach.

```
~/Developer/personal/yoga/yoga/../yoga/YGEnums.h:61:5: 
error: enumeration value 'YGErrataClassic' is out of range of flags in enumeration type 'YGErrata' [-Werror,-Wflag-enum]
   61 |     YGErrataClassic = 2147483646,
      |     ^
~/Developer/personal/yoga/yoga/../yoga/YGEnums.h:62:5: 
error: enumeration value 'YGErrataAll' is out of range of flags in enumeration type 'YGErrata' [-Werror,-Wflag-enum]
   62 |     YGErrataAll = 2147483647)
```

Do you have thoughts on the best way to handle that?